### PR TITLE
Gui Mappings

### PR DESCRIPTION
**Pull request by shedaniel**
This is my first time contributing to yarn, and I've no idea how to use Enigma, so please tell me if anything is wrong.

### CODE REVIEW

**Review by se7kn8 — APPROVED** [review]

File: mappings/net/minecraft/client/gui/container/LoomGui.mapping [hunk 1]
```diff
@@ -1,2 +1,6 @@
 CLASS cvz net/minecraft/client/gui/container/LoomGui
+	FIELD A GRAY Lawa;
+	FIELD B WHITE Lawa;
+	FIELD GRAY GRAY Lawa;
```
Comment (modmuss50) [inline mappings/net/minecraft/client/gui/container/LoomGui.mapping]: Do not map out already mapped fields.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

**Comment by asiekierka** [comment]
Pretty sure that's unmergeable at this point, so don't fuss about getting it in early - I'll go through it by hand.

File: mappings/net/minecraft/client/gui/container/LoomGui.mapping (hunk 1 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/container/LoomGui.mapping]: Alright, it is a mistake, Sorry!

**Comment by shedaniel** [comment]
Will probably create another PR with a new snapshot

**Pull request closed without merging**